HPXMLtoOpenStudio/resources/hotwater_appliances.rb

```diff
@@ -1013,6 +1011,7 @@ def self.get_dist_waste_gpd(eri_version, nbeds, has_uncond_bsmnt, cfa, ncfl, hot
     f_eff = get_fixtures_effectiveness(fixtures_all_low_flow)
 
     mw_gpd = f_eff * (o_w_gpd + s_w_gpd * wd_eff) # Eq. 4.2-11
+    mw_gpd = 0.0 if mw_gpd.is_a?(Complex) || mw_gpd < 0
```

Comment:
Would it be better to just return 0 on the first line of the method if nbeds < 0? Rather than unnecessarily go through all the code and try to handle all the possible outcomes (complex, negative). Applies to previous code change too.

Reply:
I'll try that.